docs/core/deploying/trimming/snippets/ConsoleApp1/ConsoleApp1.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <PublishTrimmed>true</PublishTrimmed>
+    <PublishAot>true</PublishAot>
+    <IsTrimmable>true</IsTrimmable>
```

Comment:
Marking the test app as `IsTrimmable` does nothing - we should remove it (`PublishTrimmed` does the same thing effectively).